mono_build_scripts_linux.sh

```
@@ -0,0 +1,19 @@
+
+if ! dotnet nuget list source | grep -q "$SOURCE_NAME"; then
+    echo "Local nuget store not found, making one in $HOME..."
+    mkdir -p "$SOURCE_PATH"
+    dotnet nuget add source "$SOURCE_PATH" --name "$SOURCE_NAME"
+fi
+
+# Build editor binary
+scons compiledb=yes dev_build=yes platform=linuxbsd target=editor module_mono_enabled=yes
+
+# Generate glue sources
+./bin/redot.linuxbsd.editor.dev.*.mono --headless --generate-mono-glue modules/mono/glue --headless --generate-mono-glue modules/mono/glue
```

Shellcheck (0.11.0)

[warning] This is a glob used as a command name. Was it supposed to be in ${..}, array, or is it missing quoting?

(SC2211)